templates/default/systemd.service.erb

```diff
@@ -10,6 +10,13 @@ ExecReload=/bin/kill -<%= @reload_signal %> $MAINPID
 KillSignal=<%= @stop_signal %>
 User=<%= @user %>
 WorkingDirectory=<%= @directory %>
+<% if node['consul']['config']['server'] %>
```

Comment:
@chuckyz / @roberteckert 

For reference, it's better practice to pass node attributes into templates rather than referencing them directly in the template.

For example:
```ruby
# recipe/foo.rb

template 'foo' do
  variables(
    isServer: node['consul']['config']['server']
  )
end

# templates/default/foo.erb

<% if @isServer -%>
SecureBits=keep-caps
AmbientCapabilities=CAP_NET_BIND_SERVICE
Capabilities=CAP_NET_BIND_SERVICE=+ep
CapabilityBoundingSet=CAP_SYSLOG CAP_NET_BIND_SERVICE
<% end -%>
```